extensions/amp-social-share/0.1/amp-social-share-config.js

```diff
@@ -63,10 +63,11 @@ const BUILTINS = dict({
     },
   },
   'email': {
-    'shareEndpoint': 'mailto:',
+    'shareEndpoint': 'mailto: ${recipient}',
```

Comment:
Nit: remove the space between `:` and `$`

Reply:
@jridgewell Tried that too. Still the example is not working.

Reply:
<img width="569" alt="screen shot 2017-09-08 at 1 52 12 am" src="https://user-images.githubusercontent.com/1635852/30183531-663a02b6-9438-11e7-8bff-e8bc7ca4c2b2.png">

Reply:
<img width="633" alt="screen shot 2017-09-08 at 1 53 23 am" src="https://user-images.githubusercontent.com/1635852/30183582-8fba5938-9438-11e7-893a-59ab588a8444.png">